tests/test_visitors/test_ast/test_loops/test_loops/test_infinite_while_loops.py

```diff
@@ -50,6 +50,36 @@ def wrapper():
     {1}
 """
 
+correct_loop_with_try1 = """
+while True:
```

Comment:
Can you please refactor this test to be `while 1`?

Reply:
Did you mean? `while 1:
Ok. I'll refactor it.